ironfish/src/node.ts

```diff
@@ -154,17 +154,18 @@ export class IronfishNode {
   }): Promise<IronfishNode> {
     logger = logger.withTag('ironfishnode')
 
+    const dataDirOrDefault = dataDir || DEFAULT_DATA_DIR
```

Comment:
I think this is my preference but I think this is simpler to just default the variable to its default value in the case that you want to set a default. We actually do use that pattern too in a lot of places, instead of creating a new variable. It also makes the diff a little simpler. What do you think?